konfera/event/views.py

```diff
@@ -74,15 +88,50 @@ def dispatch(self, *args, **kwargs):
 
     def post(self, *args, **kwargs):
         context = self.get_context_data(**kwargs)
+        notify = getattr(django_settings, 'PROPOSAL_NOTIFY', getattr(settings, 'PROPOSAL_NOTIFY'))
```

Comment:
in konfera settings you are already getting django settings (this includes also local_settings) in there is PROPOSAL_NOTIFY set, with False as a default. No need to do it here again.